bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -172,7 +177,7 @@ func (r *KubeadmConfigReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	if configOwner == nil {
 		return ctrl.Result{}, nil
 	}
-	log = log.WithValues("kind", configOwner.GetKind(), "version", configOwner.GetResourceVersion(), "name", configOwner.GetName())
+	log = log.WithValues(strings.ToLower(configOwner.GetKind()), klog.KRef(configOwner.GetNamespace(), configOwner.GetName()).String(), "version", configOwner.GetResourceVersion())
```

Comment:
/hold
We should probably not call .String based on the latest results of:
* https://github.com/kubernetes-sigs/cluster-api/pull/6190
* https://github.com/kubernetes/klog/issues/302

Reply:
Dropped the .String calls